content/10.diagnostics.md

```diff
@@ -51,15 +72,17 @@ For example, PCR-based test can be highly sensitive, but in high-throughput sett
 3. Type of specimen, as it is not clear which clinical samples are best to detect the virus [@doi:10.1038/d41587-020-00002-2].
 4. Expensive machinery, which might be present in major hospitals and/or diagnostic centers but is often not available to smaller facilities [@doi:10.1126/science.abb8400].
 5. Timing of the test, which might take up to 4 days to give results [@doi:10.1126/science.abb8400].
+6. The availability of supplies for testing, including swabs and testing media, has been limited [@doi:10.1128/JCM.00512-20].
 
 False positives are a concern with this technique, as the guide RNA can recognize other interspersed sequences on the patient’s genome, resulting in a loss of specificity.
-
 Similarly, in tests that use CRISPR, false positives can occur due to the specificity of the technique, as the guide RNA can recognise other interspersed sequences on the patient’s genome.
+As noted above, false negatives are a significant concern for several reason.
+Clinical experience has led to particular caution because negative molecular tests do not necessarily mean a patient is virus-free [doi:10.1128/JCM.00297-20]
```

Comment:
The list written by @aadattoli enumerates the limitations for PCR-based tests specifically. Right now the CRISPR limitations section is very short and therefore written in paragraph form. I'm sure there are other concerns with CRISPR we could add and perhaps it would make sense to combine these sections, but right now that's why this chunk is separate.